### DISCUSSION TITLE
How to always show 'Currently recording track'?

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#21030] [Q&A]
Is it possible to have the currently recording track show on the map by default? I can enable it in the settings and it then shows. However, the next time I start doing a track recording it again does not show on the map and I have to enable tracks in the map settings again. I may be mistaken, but I don't recall this always happening? Am I doing something wrong?

![Screenshot_20241011-183609](https://github.com/user-attachments/assets/deca5061-726a-4083-8ac2-94be22dc3cca)

**Comment by vshcherb** [comment]
It sounds like a bug, currently recording track should be persistent and I think it used to be persistent, so it's a regression. We need to double check that @yuriiurshuliak

**Reply by ERYpTION** [reply]
I played around with the settings today. Not sure what I did differently but now it finally seems the currently recording track shows every time automatically. So I guess I must simply have used a wrong setting. Probably not a bug.
